src/main/resources/templates/db_scripts/schema.json

```diff
@@ -349,6 +349,11 @@
     {
       "run": "after",
       "snippetPath": "index_dateLostItemShouldBeBilled.sql"
+    },
+    {
+      "run": "after",
+      "snippetPath": "add_staff_slips_hold_transit.sql",
+      "fromModuleVersion": "12.2.0"
```

Comment:
It looks like we're adding hold and transit slips with a separate script and setting different `fromModuleVersion` for it (`12.2.0` vs `11.1.0`). Is this how it's supposed to work? What should happen if the user upgrades to a version in between `11.1.0` and `12.2.0`? Should we consider just adding these two slips to the existing script?

Reply:
The details are now hazy, but IIRC, I asked somebody about just augmenting the existing script and was told a separate script would be better because updating an existing script would effectively prevent customers who have already run that from accessing the migration. But your comment about `fromModuleVersion` absolutely makes sense; maybe the right thing is to use multiple scripts but with the same value there?

Reply:
@alexanderkurash 

bq. Should we consider just adding these two slips to the existing script?

As I understand it, RAML Module Builder uses the association between each SQL script and a version to determine whether to execute that SQL. I don't understand the logic used completely, I believe that amending an existing script for a version that has already been deployed will mean those changes are not applied.

> What should happen if the user upgrades to a version in between 11.1.0 and 12.2.0?

I'm not sure I understand this question. We are adding the new slip definitions in version 12.2.0 as that is the next intended release. Are you asking if we should be also back porting this change to previous releases?

Reply:
Also, `12.2.0` has already been released, which I believe means that the `fromModuleVersion` for this script needs to be `12.3.0` for this to be executed on existing systems.

Reply:
Ok, second script makes sense to me now.

> I'm not sure I understand this question. We are adding the new slip definitions in version 12.2.0 as that is the next intended release. Are you asking if we should be also back porting this change to previous releases?

Yes, this is what I meant. I believe we've added those slips a long time ago (in 2019), but didn't add them to the migration script. If we set `fromModuleVersion` to the latest version, existing systems would need to be updated to this version in order to get these slips. I don't know, maybe this is a good thing if no one is planning to update to older versions.

> Also, 12.2.0 has already been released, which I believe means that the fromModuleVersion for this script needs to be 12.3.0 for this to be executed on existing systems.

Should it be 12.2.1, considering that CIRCSTORE-273 is marked as R1 bugfix?

Reply:
> I don't know, maybe this is a good thing if no one is planning to update to older versions.

If folks want an update to the older distributions of FOLIO, then I think they will need to ask for this change to be back ported.

> Should it be 12.2.1, considering that CIRCSTORE-273 is marked as R1 bugfix?

I don't have an authoritative answer for you. Maybe the RAML Module Builder folks can advise on that.

My personal preference is for the script to be marked as _from_ `12.3.0` on the mainline and `12.2.1` on the patch release branch. This is due to `12.2.1` not being a legitimate version for the mainline and so this configuration could be confusing (and I personally find this area really confusing). This does mean that the script needs to be repeatable, as folks can upgrade from `12.2.1` to `12.3.0`, and it does mean the back port needs to change the code.